package-lock.json

```diff
@@ -17926,7 +17926,6 @@
       "version": "3.0.0",
       "resolved": "https://registry.npmjs.org/strip-css-comments/-/strip-css-comments-3.0.0.tgz",
       "integrity": "sha1-elYl7/iisibPiUehElTaluE9rok=",
-      "dev": true,
```

Comment:
I'm not sure why this got removed, but the package.json seems fine and we don't publish the lock anyway, so I will just ignore it...

Reply:
@thgreasi `scope-css` requires `strip-css-comments`, so that is probably why. But yes, it is irrelevant.